README.md

```diff
@@ -93,6 +100,10 @@ Nginx Configuration:
 * `NGINX_MAX_CLIENT_BODY_SIZE`: Configures the max client request body size of nginx. _(Default: `0`, Disables checking of client request body size.)_
 * `NGINX_STATUS_PATH`: Change the nginx status path that returns 200 on the default server. (Default: `/dispatcher/status`)
 * `PORT`: This is the port that nginx will listen on _(Default: `80`)_
+* `SSL_PORT`: This is the SSL port nginx will listen on for ssl enabled hosts. _(Default: `443`)_
+* `SSL_CERT`: Path to a ssl certificate used for the default ssl nginx server. Enables SSL support.
+* `SSL_KEY`: Path to a ssl certificate key used for the default ssl nginx server.
+* `SSL_CERT_DIR`: Path to write namespace host's certificates to. _(Default: `/etc/nginx/ssl`)_
```

Comment:
Why is this necessary?  If we own the container, can't we just put them where we want using whatever convention we want?  We don't let the user dictate where the logs are written for example.